src/core/server/http/http_server.mocks.ts

```diff
@@ -61,10 +63,11 @@ function createKibanaRequestMock({
       method,
       url: {
         path,
+        pathname: path,
```

Comment:
__note:__ theoretically we should separate `path` and `pathname`, but I thought it'd be okay to not do that for mock.